main.go

```diff
@@ -79,13 +78,10 @@ func main() {
 		Scheme:                 scheme,
 		HealthProbeBindAddress: probeAddr,
 		LeaderElection:         enableLeaderElection,
-		LeaderElectionID:       "lca.ran.openshift.io",
+		LeaderElectionID:       "lca.openshift.io",
```

Comment:
Can go in another PR but need to update for the correct values for SNO

Reply:
There are rules for this on SNO? The ptp operator uses `ptp.openshift.io`

Reply:
An example here,  https://github.com/openshift/ptp-operator/blob/release-4.13/main.go#L57